File: Modul 3/Soal 4/PRAK304-2410817320005-NadiaFelicia.c
#include <stdio.h>

int main() {
    int rafa;
    scanf("%d", &rafa);

    if (rafa == 0) {
        printf("Nol");
    }
    else if (rafa == 10 || rafa >= 20 && rafa <=99) {
        printf("Puluhan");
    }
    else if (rafa >= 1 && rafa <= 9) {
        printf("Satuan");
    }
    else if (rafa >= 11 && rafa <= 19) {
        printf("Belasan");
    }
    else {
        printf("Anda Menginput Melebihi Limit Bilangan");
    }
    }

File: Modul-3/Soal 4/PRAK304-2410817320005-NadiaFelicia.py
rafa = input()
rafa = int(rafa)

if rafa == 0:
    print("nol")
elif rafa >= 1 and rafa <=9:
    print("Satuan")
elif rafa >= 11 and rafa <=19:
    print("Belasan")
elif rafa >= 20 and rafa <=99:
    print("Puluhan")
else:
    print("Anda Menginput Melebihi Limit Bilangan")